From 602a1dc746070712f571e422ee18b6226aa98421 Mon Sep 17 00:00:00 2001
From: Hong Hao <oahong@gmail.com>
Date: Tue, 27 Jun 2017 23:07:37 +0800
Subject: [PATCH] Use GNUInstallDirs

---
 src/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3997e0e..ca08f0b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -66,7 +66,7 @@ include(LXQtTranslateDesktop)
 
 file(GLOB desktop_files_in ../data/*.desktop.in)
 lxqt_translate_desktop(desktop_files SOURCES ${desktop_files_in})
-install(FILES ${desktop_files} DESTINATION share/applications)
+install(FILES ${desktop_files} DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/applications")
 
 add_executable(lximage-qt
     ${lximage-qt_SRCS}
@@ -76,7 +76,7 @@ add_executable(lximage-qt
 )
 
 add_definitions(
-    -DLXIMAGE_DATA_DIR="${CMAKE_INSTALL_PREFIX}/share/lximage-qt"
+    -DLXIMAGE_DATA_DIR="${CMAKE_INSTALL_DATAROOTDIR}/lximage-qt"
     -DLXIMAGE_VERSION="${LXIMAGE_VERSION}"
 )
 
@@ -90,4 +90,4 @@ target_link_libraries(lximage-qt
     ${XFIXES_LIBRARIES}
 )
 
-install(TARGETS lximage-qt RUNTIME DESTINATION bin)
+install(TARGETS lximage-qt RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
-- 
2.13.2

