From 151ef16fc9f889d8b4f6e8bbdb774c923fd93757 Mon Sep 17 00:00:00 2001
From: Alf Gaida <agaida@siduction.org>
Date: Sun, 22 Jun 2014 21:27:35 +0200
Subject: [PATCH 4/4] simplify QT switching, use ENV

---
 CMakeLists.txt | 13 ++-----------
 1 file changed, 2 insertions(+), 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b528641..a4d64bd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,15 +2,8 @@ cmake_minimum_required(VERSION 2.6)
 
 project(libsysstat)
 
-option(USE_QT5 "Build with Qt5." OFF)
-if (USE_QT5 AND USE_QT4)
-    message(FATAL "USE_QT4 and USE_QT5 both set. Please choose *only* one Qt version")
-elseif(USE_QT5)
-    set(USE_QT4 OFF)
-else()
-    set(USE_QT4 ON)
-endif()
-
+# Support different versions of Qt
+option(USE_QT5 "Build with Qt5." $ENV{USE_QT5})
 
 # Set default installation paths
 set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE PATH "Installation path for libraries")
@@ -37,12 +30,10 @@ if (USE_QT5)
     find_package(Qt5Core REQUIRED QUIET)
     find_package(Qt5LinguistTools REQUIRED QUIET)
     set(QT_VERSION_SUFFIX "-qt5")
-    message(STATUS "Building with Qt${Qt5Core_VERSION_STRING}")
 else()
     find_package(Qt4 REQUIRED)
     include(${QT_USE_FILE})
     set(QT_VERSION_SUFFIX "")
-    message(STATUS "Building with Qt${QTVERSION}")
 endif()
 
 set(SYSSTAT_LIBRARY_NAME "sysstat${QT_VERSION_SUFFIX}")
-- 
2.1.2

